index.js

```diff
@@ -6,17 +6,19 @@ const artifactClient = new DefaultArtifactClient();
 const { exec } = require("child_process");
 const util = require("node:util");
 const semver = require("semver");
-
 const { Octokit } = require("@octokit/rest");
-const octokit = new Octokit();
+
 const manifestOwner = "manifest-cyber";
 const manifestRepo = "cli";
 const manifestBinary = "manifest-cli";
-const jqOwner = "jqlang";
-const jqRepo = "jq";
 const jqBinary = "jq";
 const tmpPath = "/tmp";
 
+const githubApiToken = core('githubToken') || core('githubtoken') || process.env.GITHUB_TOKEN || undefined;
```

Comment:
```suggestion
const githubApiToken = core('githubToken') || process.env.GITHUB_TOKEN || undefined;
```

I think @marcelolima381 already flagged this though

Reply:
oh wait, ignore me, noticed the casing diff